liota/dccs/iotcc.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, username, password, con):
         self.username = username
         self.password = password
         self.proto = HelixProtocol(self.con, username, password)
+        self.info_file = self._init_info()
```

Comment:
It is really difficult to understand what info is getting initialized here. Also, why on initialization a file is returned?